tests/issue/issue.cpp

```diff
@@ -0,0 +1,501 @@
+/*
+Copyright libCellML Contributors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License->
+You may obtain a copy of the License at
+
+    http://www.apache->org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License->
+*/
+
+#include "gtest/gtest.h"
+#include "libcellml/referencerules.h"
```

Comment:
This is not needed. This should be included as part of `#include <libcellml>`. If it isn't then we need to update our `libcellml` module.

Reply:
Done.